agent/engine/docker_task_engine_linux_test.go

```diff
@@ -58,7 +58,6 @@ import (
 	"github.com/aws/amazon-ecs-agent/ecs-agent/credentials"
 	"github.com/aws/amazon-ecs-agent/ecs-agent/netlib/model/appmesh"
 	ni "github.com/aws/amazon-ecs-agent/ecs-agent/netlib/model/networkinterface"
-
```

Comment:
nit: this newline was intentionally added, to distinguish between internal and external 3rd party imports.

Reply:
Ah right. Fixed it.